doc/README.md

```diff
@@ -0,0 +1,21 @@
+# pyglmnet documentation
+
+See full documentation page [here](http://pavanramkumar.github.io/pyglmnet/).
```

Comment:
Totally agree @jasmainak. @pavanramkumar, can you put the documentation link in upper tab of the repo?